packages/plugins/src/plugins/morpho/lendingFetcher.ts

```diff
@@ -0,0 +1,141 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+import { Fetcher, FetcherExecutor } from '../../Fetcher';
+import { morphoMarketsCachePrefix, platformId } from './constants';
+import { getEvmClient } from '../../utils/clients';
+import { ElementRegistry } from '../../utils/elementbuilder/ElementRegistry';
+import { Cache } from '../../Cache';
+import { MorphoMarketRes } from './types';
+import { morphoContractABI } from './utils/abis';
+
+const morphoContract = '0xbbbbbbbbbb9cc5e90e3b3af64bdaf62c37eeffcb';
```

Comment:
Aren't we supposed to used the checksummed version/representation ?

```suggestion
const morphoContract = '0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb';
```